templates/security/cve/filters/status-filter.html

```diff
@@ -0,0 +1,27 @@
+<form>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="fixed" class="p-checkbox__input" />
```

Comment:
On the current live page there are 8 status filters but only 6 here, is this accurate?